app/javascript/components/pages/key-performance-indicators/action-creators.js

```diff
@@ -0,0 +1,10 @@
+import * as Actions from "./actions";
+
+export const fetchNumberOfCases = () => async dispatch => {
```

Comment:
I would comment this action out and create another one with the same name. Create a action in action.js `KeyPerformanceIndicators/NUMBER_OF_CASES_SUCCESS` This new action creator would just have a type of that new action and a payload of your mocked data in the selector below. That is the way we mocked data for components, so we can make sure our components work correctly with immutablejs. Also the selector can do the actual selecting from redux state.